### PR TITLE
Get everything up-to-date in preparation for the new decade

### PR DESCRIPTION
**Pull request by scudco**
* Add Ruby 2.7 and Rails 6.0 to travis build
* Remove support for Rails <5.2
* Remove support for Ruby <2.5

### CODE REVIEW

**Comment by olleolleolle** [comment]
Perhaps this commit could be in here, too? We are not running rbx in the matrix.

https://github.com/stefanpenner/country_select/pull/179/commits/54f3e26e54903b2244d90dbfe81a1cbd99541f11

**Comment by scudco** [comment]
just added your changes here with a co-author tag, hope that works for you

**Review by olleolleolle — COMMENTED** [review]
Awesome work, and big progress.

I made inline notes, perhaps they jog your memory in some way.

Have a nice day!

File: Gemfile.lock [hunk 1]
```diff
@@ -158,6 +164,7 @@ GEM
       rubysl-tmpdir (~> 2.0)
       rubysl-tsort (~> 2.0)
       rubysl-un (~> 2.0)
+      rubysl-unicode_normalize (~> 2.0)
```
[the remark this answers is not in the document]
Comment (scudco) [reply Gemfile.lock]: It was still in the Gemfile. good find!

**Pull request merged**